client/app/pods/components/user-avatar/component.coffee

```diff
@@ -0,0 +1,18 @@
+`import Ember from 'ember'`
+`import FileUploadMixin from 'tahi/mixins/file-upload'`
+`import ValidationErrorsMixin from 'tahi/mixins/validation-errors'`
+`import Utils from 'tahi/services/utils'`
+
+UserAvatarComponent = Ember.Component.extend FileUploadMixin, ValidationErrorsMixin,
+  errorText: ""
+
+  avatarUploadUrl: ( ->
+    "/users/#{@get('model.id')}/update_avatar"
```

Comment:
It's probably better to use contextual entity here, so if you're passing in `user`, use that instead of `model`.